pkg/cluster/graph/manager.go

```diff
@@ -42,33 +41,31 @@ func NewManager(log *logrus.Entry, aead encryption.AEAD, storage storage.Manager
 func (m *manager) Exists(ctx context.Context, resourceGroup, account string) (bool, error) {
 	m.log.Print("checking if graph exists")
 
-	blobService, err := m.storage.BlobService(ctx, resourceGroup, account, mgmtstorage.Permissions("r"), mgmtstorage.SignedResourceTypesO)
+	blobService, err := m.storage.BlobService(ctx, resourceGroup, account, armstorage.Permissions("r"), armstorage.SignedResourceTypesO)
 	if err != nil {
 		return false, err
 	}
 
-	aro := blobService.GetContainerReference("aro")
-	return aro.GetBlobReference("graph").Exists()
+	return blobService.BlobExists(ctx, "aro", "graph")
 }
 
 // Load() should not be implemented: use LoadPersisted
 
 func (m *manager) Save(ctx context.Context, resourceGroup, account string, g Graph) error {
 	m.log.Print("save graph")
 
-	blobService, err := m.storage.BlobService(ctx, resourceGroup, account, mgmtstorage.Permissions("cw"), mgmtstorage.SignedResourceTypesO)
+	blobService, err := m.storage.BlobService(ctx, resourceGroup, account, armstorage.Permissions("cw"), armstorage.SignedResourceTypesO)
 	if err != nil {
 		return err
 	}
 
 	bootstrap := g.Get(&bootstrap.Bootstrap{}).(*bootstrap.Bootstrap)
-	bootstrapIgn := blobService.GetContainerReference("ignition").GetBlobReference("bootstrap.ign")
-	err = bootstrapIgn.CreateBlockBlobFromReader(bytes.NewReader(bootstrap.File.Data), nil)
+
+	_, err = blobService.UploadBuffer(ctx, "ignition", "bootstrap.ign", bootstrap.File.Data, nil)
```

Comment:
can we extract magic strings?

- `aro`
- `graph`
- `ignition`
- `bootstrap.ign`

Reply:
Created consts.